template-only-bin/install-app.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+# -----------------------------------------------------------------------------
+# This script installs infrastructure for an application.
+# It is called by other scripts.
+#
+# Positional parameters:
+#   APP_NAME (required) - the name for the application, use kebab-case
+#   DST_PREFIX (optional) - the directory that the application should be installed in
+#     Defaults to "" (the current directory). If directory is supplied, must contain a
+#     trailing slash.
+# -----------------------------------------------------------------------------
+set -euo pipefail
+
+APP_NAME=$1
+DST_PREFIX=${2:-""}
+
+echo "Creating a terraform module for application: $APP_NAME..."
+cp -r template-infra/infra/app "${DST_PREFIX}infra/$APP_NAME"
+
+# Helper to get the correct sed -i behavior for both GNU sed and BSD sed (installed by default on macOS)
+# Hat tip: https://stackoverflow.com/a/38595160
+sedi () {
+  if sed --version >/dev/null 2>&1; then
+    sed -i -- "$@"
+  else
+    sed -i "" "$@"
+  fi
+}
```

Comment:
is this necessary? we already use sed in the template in a lot of places and i don't recall needing this.